### PR TITLE
Frontend Basic Setup

### PR DESCRIPTION
**Pull request by muhammad-ahmed02**
This is just a template based on Next.js and tailwind css

### CODE REVIEW

File: frontend/app/LICENSE
Comment (RusabKhan) [inline frontend/app/LICENSE:1]: this file is not required, you can delete this.

File: frontend/app/README.md
Comment (RusabKhan) [inline frontend/app/README.md:1]: No needed delete this file

File: frontend/dockerfile
Comment (RusabKhan) [inline frontend/dockerfile:1]: This dockerfile is now useless. It was used for the streamlit frontend. Create a new docker file.

**Comment by muhammad-ahmed02** [comment]
Commited new changes please review them

**Pull request closed without merging**